web/web.go

```diff
@@ -176,7 +177,7 @@ func getTemplate(name string, pathPrefix string) (*template.Template, error) {
 			return lset
 		},
 		"globalURL": func(url string) string {
-			hostname, err := os.Hostname()
+			hostname, err := getServerHostname()
```

Comment:
`hostname` clashes with `hostname()`, doesn't it? I have to admit that my Go experience is quite short :-)
Making it uppercase would fix it, but as far as I know that would make the function public. Would you be ok with `Hostname()`?